woocommerce/woocommerce-ios
# Fix 'contentEdgeInsets' was deprecated in iOS 15.0

### Description
By updating the minimum release version to iOS 15 recently, we've seen some deprecation warnings. This PR fixes the case for `'contentEdgeInsets' was deprecated in iOS 15.0: This property is ignored when using UIButtonConfiguration`, for some cases.

### Changes
1. Updated the following classes to use `UIButton.Configuration` contentInsets
- [x] UIButton+Helpers
- [x] CustomerNoteTableViewCell
- [x] EmptyStateViewController
- [x] FilteredOrdersHeaderBar
- [x] SurveySubmittedViewController
- [x] VerticalButton

2. Added titleLabel to subview for `ButtonActivityIndicator` due to an edge case encountered while testing: Using the new configuration approach made the app crash (and test fail) when we went through the card present payment flow if the titleLabel required to use multiline (pe: small screens, different languages, bigger system accessibility fonts, ... ). This is because we are calling [pinSubViewsToAllEdgeMargins(_:)](https://github.com/woocommerce/woocommerce-ios/blob/2f718b2bb8f848f24c194aa02193ed79f5747ee1/WooCommerce/Classes/Extensions/UIButton%2BHelpers.swift#L150) while label and button do not share the same ancestor for that specific case, getting this error:

```
'Unable to activate constraint with anchors <NSLayoutXAxisAnchor:0x600003f55700 "UILayoutGuide:0x600001334460'UIViewLayoutMarginsGuide'.leading"> and <NSLayoutXAxisAnchor:0x600003f55740 "UILabel:0x155d4f9a0.leading"> because they have no common ancestor.  
Does the constraint or its anchors reference items in different view hierarchies?  That's illegal.'
```

### Testing instructions
Go to the following views and confirm that buttons appear correctly:
1. Go to Orders > Filter
2. Orders > For different Orders confirm that primary ("Mark Order as Completed") and secondary buttons ( "Issue Refund") appear correctly, as well as the "Add Customer Note" button
3. Menu > Reviews > Confirm that spam, trash, approve, and reply buttons
4. Menu > Settings > Send Feedback > Confirm the "Contact us here" button after feedback is sent

### Screenshots
|  UIButton+Helpers | CustomerNoteTableViewCell | FilteredOrdersHeaderBar 
|---|---|---|
|   |   |   |
| <img width=400 src="https://user-images.githubusercontent.com/3812076/201839775-8c123f94-a062-4fca-99a0-39a6c8dcdf3b.png"> | <img width=400 src="https://user-images.githubusercontent.com/3812076/201846687-d411edf4-4a81-492f-8057-f6968ae193b6.png"> |  <img width=400 src="https://user-images.githubusercontent.com/3812076/202060180-314a98e5-a1f8-42b0-a8b6-411abc9c537b.png"> | 

| SurveySubmittedViewController  | VerticalButton  | |
|---|---|---|
|  <img width=400 src="https://user-images.githubusercontent.com/3812076/201845681-eb4b7fd5-351c-4b33-9cbb-06a6d0f8b8aa.png"> | <img width=400 src="https://user-images.githubusercontent.com/3812076/201839651-8bb92e70-04c2-42ba-8845-e063d849e423.png"> | |
